build.sbt

```diff
@@ -107,7 +107,7 @@ lazy val core = project
       "com.fasterxml.jackson.core"             % "jackson-core"                 % V.jackson,
       "com.fasterxml.jackson.core"             % "jackson-databind"             % V.`jackson-databind`,
       "dev.zio"                               %% "zio-interop-cats"             % V.`zio-interop`,
-      "dev.zio"                               %% "zio-kafka"                    % V.`zio-kafka`,
+      "dev.zio"                               %% "zio-kafka"                    % V.`zio-kafka` excludeAll ("org.apache.kafka", "kafka-clients"),
```

Comment:
this is to downgrade kafka-client to the explicit version we can support, due to embedded-kafka-schema-registry non binary compatible with latest embedded-kafka